tests/rabbitmq.nix

```diff
@@ -1,29 +1,37 @@
 import ./make-test-python.nix ({ lib, pkgs, testlib, ... }:
 let
   # Default IP automatically assigned in NixOS tests. Must not be changed here.
-  ipv4 = "192.168.1.1";
+  ipv4 = "192.168.3.1";
+  nodeDefaults = {
+    imports = [ (testlib.fcConfig { }) /*../nixos ../nixos/roles*/ ];
```

Comment:
`testlib.fcConfig` takes an optional `id` argument, defaulting to 1. This is used to derive node's IP addresses, MACs and other network-related config.

The current re-usage of `nodeDefaults` causes both nodes to have equal addresses. This currently happens to work, as we do not test any network interactions between individual nodes, but nonetheless isn't really clean and *might* cause issues at some point. => I prefer to have different IDs per node.